packages/web/src/components/user-badges/TierExplainerDrawer.tsx

```diff
@@ -1,18 +1,37 @@
-import { useCallback } from 'react'
+import { ReactElement, useCallback } from 'react'
+
+import { BadgeTier } from '@audius/common/models'
+import { Nullable } from '@audius/common/utils'
+import {
+  IconTokenBronze,
+  IconTokenGold,
+  IconTokenPlatinum,
+  IconTokenSilver
+} from '@audius/harmony'
 
 import { useModalState } from 'common/hooks/useModalState'
 import Drawer from 'components/drawer/Drawer'
 import { useProfileTier } from 'hooks/wallet'
-import {
-  audioTierMapSvg,
-  TierLevel,
-  TierNumber
-} from 'pages/audio-rewards-page/Tiers'
+import { TierLevel, TierNumber } from 'pages/audio-rewards-page/Tiers'
 
 import { BadgeTierText } from './ProfilePageBadge'
 import styles from './TierExplainerDrawer.module.css'
 import { messages } from './TierExplainerModal'
 
+const BADGE_SIZE = 108
```

Comment:
nit: could be shared constant.

Reply:
i made this not shared specifically b/c the size is layout dependent - originally these were all imported from the one pages component, but that made this really hard to change because i broke a few different layouts in the app trying to upgrade this one